src/types/getters.ts

```diff
@@ -43,6 +53,9 @@ export function getPropertyOfType(
 	return type.getProperties().find((s) => s.escapedName === name);
 }
 
+/**
+ * @category Types - Getters
+ */
 export function getWellKnownSymbolPropertyOfType(
```

Comment:
@JoshuaKGoldberg would you be able to write a quick doc for this function?
I'm not sure how to word the "WellKnownSymbol" thing?

Reply:
How about _`* Retrieves a type symbol corresponding to a well-known string name.`_ for now?